portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
```

Comment:
Love the separate data structure! It's super Object Oriented Programming-esque, and I love it ❤️